### PR TITLE
Splatting uses colors stored in a texture instead of vertex buffer

### PR DESCRIPTION
**Pull request by mvaligursky**
Additional components will be moved to a texture in a separate PR. The color will be also removed from VB in a separate PR.

### CODE REVIEW

File: src/splat-resource.ts [hunk 1]
```diff
@@ -334,6 +386,27 @@ class SplatResource extends ContainerResource {
 
     }
 
+    evalTextureSize(count: number) : Vec2 {
+        const width = Math.ceil(Math.sqrt(count));
```
[the remark this answers is not in the document]
Comment (mvaligursky) [reply src/splat-resource.ts:390]: for what raason?

[the remark this answers is not in the document]
Comment (mvaligursky) [reply src/splat-resource.ts:390]: this is the code the engine uses for morphing .. no power of 2

[the remark this answers is not in the document]
Comment (mvaligursky) [reply src/splat-resource.ts:390]: as long as you don't use mipmaps and use clamp addressing, non-power of two are fine on WebGL1

[the remark this answers is not in the document]
Comment (mvaligursky) [reply src/splat-resource.ts:390]: clustered lighting is a good example

**Review by slimbuck — APPROVED** [review]

**Pull request merged**